OneCommunityGlobal/HighestGoodNetworkApp
# Common folder prettier fix

# Description
This PR is for Prettier fixing the `src\components\common` folder

## Main changes explained:
- `.prettierignore`: remove the common so Prettier rules can on it
- other files: apply changes to match the Prettier rules

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. Clear site data/cache
4. log as admin user
5. make sure everything is file

## Screenshots or videos of changes:

## Note:
